### DISCUSSION TITLE
Tools to convert Json Schema to UML diagrams or any way of good visual representation

### DISCUSSION THREAD

**Discussion by mathan-mohan91** [json-schema-org/community#642] [Q&A]
Hey, 
Are there any tools out there to convert Json Schema to UML diagrams? There's a tool in intellij which converts a single file to a UML representation but since I have references to multiple files it's not showing up fine.

**Comment by pvlasov** [comment]
Hello, 

I've created a solution, not a tool, for generating JSON schema documentation with visualizations: [JSON Schema documentation generator](https://github.com/orgs/json-schema-org/discussions/910).
Currently it works on a single schema, but can be extended to generate cross-referenced sites.

Best regards, Pavel.
